Tests/DatadogTests/Datadog/Core/Persistence/LoggingIOBenchmarkTests.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  LoggingIOBenchmarkTests.swift
+//  DatadogTests
+//
+//  Created by Mert Buran on 15/04/2020.
+//  Copyright © 2020 Datadog. All rights reserved.
+//
+
+import XCTest
+@testable import Datadog
+
+class LoggingIOBenchmarkTests: XCTestCase {
+    // swiftlint:disable implicitly_unwrapped_optional
+    private var queue: DispatchQueue!
+    private var directory: Directory!
+    private var strategy: LogsPersistenceStrategy!
+    // swiftlint:enable implicitly_unwrapped_optional
+
+    override func setUp() {
+        super.setUp()
+        self.queue = DispatchQueue(label: "com.datadoghq.benchmark-logs-io", target: .global(qos: .utility))
+        self.directory = try! Directory(withSubdirectoryPath: "logging-benchmark")
+        self.strategy = .default(in: directory, using: SystemDateProvider(), readWriteQueue: queue)
+        XCTAssertTrue(try! directory.files().count == 0)
+    }
+
+    override func tearDown() {
+        try! FileManager.default.removeItem(at: directory.url)
```

Comment:
Isn't this just
```suggestion
        self.directory.delete()
```
now?